lib/hammer_cli_katello/hostgroup_extensions.rb

```diff
@@ -20,6 +20,16 @@ def self.included(base)
         _('Organization name to search by'), attribute_name: :option_organization_name
       base.option '--query-organization-label', 'ORGANIZATION_LABEL',
         _('Organization label to search by'), attribute_name: :option_organization_label
+
+      base.validate_options do
+        organization_options = [
```

Comment:
The `OrganizationOptions` module isn't included here because the org options are `--query-organization[-id|-label]` rather than `--organization[-id|-label]`. In addition, this is an array of the  `attribute_name` of the options, not a hash of options.